public/js/master.js

```diff
@@ -307,7 +310,13 @@ document.addEventListener("DOMContentLoaded", async() => {
                     flagsElement.style.display = "none";
                 }
                 else {
-                    flagsElement.textContent = textContent;
+                    [...textContent].forEach((icons) => {
+                        if (icons !== " ") {
+                            flagsElement.innerHTML += `<div class="tooltip">${icons}
```

Comment:
should be a bit better to construct the string in the loop and call `.innerHTML` once to avoid parsing the html many times